lib/engine-config-from-meta.js

```diff
@@ -6,7 +6,9 @@ try {
   config = JSON.parse(unescape(rawConfig));
 }
 catch(err) {
-  throw new Error('Could not read config from meta tag with name "' + metaName + '" due to error: ' + err);
+  // in the case where there is no meta tag we can default to an empty pojo
+  // instead of throwing an error
+  config = {};
```

Comment:
Lets tweak this to separate the `document.querySelector` from the `JSON.parse(unescape` bit. I'm thinking something like:

```js
var config = {};
var metaElement = document.querySelector('meta[name="{{MODULE_PREFIX}}/config/environment"]');

if (metaElement) {
  try {
    config = JSON.parse(unescape(metaElement.getAttribute('content')));
  } catch (error) {
    throw new Error('Could not read config from meta tag for `{{MODULE_PREFIX}}` due to an error:\n' + error); 
  }
}

export default config;
```

So basically:

* an invalidly composed meta element is still going to be an error (this is important IMHO)
* having no meta element at all is perfectly fine

Reply:
I agree, this is better